packages/medusa/src/models/product.ts

```diff
@@ -248,98 +275,118 @@ export class Product extends SoftDeletableEntity {
  *     type: array
  *     items:
  *       $ref: "#/components/schemas/ProductVariant"
+ *   categories:
+ *     description: Available if the relation `category_children` are expanded.
+ *     type: array
+ *     items:
+ *       $ref: "#/components/schemas/ProductCategory"
```

Comment:
`categories` is available twice (an old definition is available below)

Reply:
😅 Fixed in 4203dc61870fc6042e2acef3beb0481f5e3d2aa6